test/python/test_gravity_model.py

```diff
@@ -171,3 +171,39 @@ def test_polyhedral_evaluable_pickle(
     acceleration = np.array([result[1] for result in sol])
     np.testing.assert_array_almost_equal(potential, expected_potential)
     np.testing.assert_array_almost_equal(acceleration, expected_acceleration)
+
+
+def test_polyhedron_metric() -> None:
+    """Tests the metric conversion/ options
+    """
+    computation_point = np.array([1.0, 0.0, 1.0])
+    gravity_constant = 6.67430e-11
```

Comment:
hypothetically, did we ever test for another gravity constant? Just as a sanity check

Reply:
No, the gravity constant is hardcoded as `6.67430e-11` $m^3/(kg*s^2)$ and is not interchangeable.

It's part of this Pull Request to change this (to a degree). The metric is now configurable to `km` instead of `m`.
Also, a `unitless` calculation without the multiplication of the gravity constant is configurable.